portfolio/src/main/webapp/script.js

```diff
@@ -14,7 +14,10 @@
 
 
 const pictures = ["images/photos/proj1.jpg", "images/photos/proj2.jpg", "images/photos/proj3.jpg"]
+const mePics = ["url('images/photos/me2.jpeg')", "url('images/photos/me3.jpeg')", "url('images/photos/me1.jpeg')"]
+
 var currentIndex = 0;
+var currentIndex2 = 0;
```

Comment:
Not terribly descriptive names, as soon as you get more than one of something it's often more useful to name it "currentProjectIndex" and "currentHeadshotIndex" or something to be a bit clearer. 

In this case I wasn't sure at first if they were both used together or not, and had to read the code carefully to make sure a 2 wasn't tacked on anywhere in the first function.

Also, use let here instead of var. Var is essentially deprecated (it has a whole host of scoping issues).